packages/daemon/src/app/controller/plugin.ts

```diff
@@ -48,4 +47,35 @@ export class PluginController {
     });
     successRes(this.ctx, { data: plugins });
   }
+
+  @post('/upload')
```

Comment:
```suggestion
  @post('/package')
```

Reply:
This function is not an action about packaging, what's your meaning?

Reply:
Post a "upload" is confusing and incorrect grammar.

Reply:
Not `post a "upload"`, just `upload`. I prefer the api name should be verb not noun.

Reply:
We declared the "post" in the decorator.